src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,40 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
+1.11.0.dev0 (09/14/2018)
+------------------------
+
```

Comment:
There might be some additional entries here... I'd be fine with "carefully" adding them and landing without waiting for another round of CI.

Reply:
In fact, if this goes green, I'll likely do that for you in order to draw the line in the sand that you can release from later.